vulcan/lib/client/jsx/components/workflow/steps/step.jsx

```diff
@@ -6,7 +6,7 @@ import StepName from './step_name';
 export default function Step({step, index, onClick}) {
   const {status, pathIndex} = useContext(VulcanContext);
 
-  if (!status || !pathIndex) return null;
+  if (!status || null === pathIndex) return null;
```

Comment:
minor style nit, but what about `pathIndex == null` ?  Double equals null is safer because it catches both undefined and null.